addons/mail/static/src/models/thread/thread.js

```diff
@@ -577,26 +577,27 @@ function factory(dependencies) {
         /**
          * Mark the specified conversation as read/seen.
          *
-         * @param {integer} messageId the message to be considered as last seen
+         * @param {float} messageId the message to be considered as last seen
          */
         async markAsSeen(messageId) {
+            // commands have fake message id that is not integer
+            const lastMessageId = Math.floor(messageId);
```

Comment:
Would clarify in parameter command that genuine messages have integer as id, and transient messages have decimals. Transient messages do not exist in server, which is why actual mark as seen on conversation for server must be applied to last genuine messages.